content/documentation/staging/faq/istio-component-status/add-extra-component.adoc

```diff
@@ -0,0 +1,9 @@
+---
+title: How can I add one component to the list
+weight: 4
+---
+:icons: font
+
+If you are interested in adding one more component to the Istio Component Status tooltip, you have the option to add one new component into the https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml#L484[Kiali CR].
```

Comment:
Not sure a line number is a good idea in a doc link as that yaml changes fairly frequently.